test/testdrive/select.td

```diff
@@ -0,0 +1,17 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+> CREATE VIEW data (a, b) AS VALUES (1, 1), (2, 1), (3, 1), (1, 2)
```

Comment:
might be nice to also add a test showing that having a as of in a subquery fails?

Reply:
ah this didn't load until the page refreshed on merge, will do

Reply:
no worries! sorry for my lag in reviewing